Amethyst/Managers/WindowManager.swift

```diff
@@ -85,24 +85,11 @@ final class WindowManager<Application: ApplicationType>: NSObject, Codable {
     }
 
     func screenManager(screen: Screen) -> ScreenManager<WindowManager<Application>> {
-        let screenManager = ScreenManager<WindowManager<Application>>(
+        return ScreenManager<WindowManager<Application>>(
```

Comment:
This whole function is only used in one place, I think, so we can probably get rid of it.

Reply:
To do this, I needed to replace userConfiguration with UserConfiguration.shared - hope that's all right - I don't think any tests rely on injecting a special configuration into ScreenManager.